metrics_utility/management/validation.py

```diff
@@ -4,6 +4,37 @@
 from metrics_utility.exceptions import MissingRequiredEnvVar
 
 
+# Constants for valid values
+VALID_REPORT_TYPES = {'CCSP', 'CCSPv2', 'RENEWAL_GUIDANCE'}
+VALID_SHEETS = {
+    'CCSP': {
+        'ccsp_summary',
+        'managed_nodes',
+        'indirectly_managed_nodes',
+        'inventory_scope',
+        'usage_by_collections',
+        'usage_by_roles',
+        'usage_by_modules',
+        'usage_by_organizations',
+    },
+    'CCSPv2': {
+        'ccsp_summary',
+        'jobs',
+        'managed_nodes',
+        'indirectly_managed_nodes',
+        'inventory_scope',
+        'usage_by_organizations',
+        'usage_by_collections',
+        'usage_by_roles',
+        'usage_by_modules',
+        'managed_nodes_by_organization',
+        'data_collection_status',
+    },
+}
+VALID_COLLECTORS = {'main_host', 'main_jobevent', 'main_indirectmanagednodeaudit'}
+VALID_SHIP_TARGET = {'directory', 's3', 'crc'}
```

Comment:
```suggestion
VALID_SHIP_TARGET = {'directory', 's3', 'crc', 'controller_db'}
```

(though `crc` is only available in gather, not build_report)